packages/contracts/contracts/L1/OptimismPortal.sol

```diff
@@ -0,0 +1,121 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.10;
+
+/* Inherited Imports */
+import { DepositFeed } from "./DepositFeed.sol";
+
+/* Interactions Imports */
+import { L2OutputOracle } from "./L2OutputOracle.sol";
+
+/* Library Imports */
+import { WithdrawalVerifier } from "./Lib_WithdrawalVerifier.sol";
+
+/**
+ * @title OptimismPortal
+ * @notice The OptimismPortal is a contract on L1 used to deposit and withdraw between L2 and L1.
+ */
+contract OptimismPortal is DepositFeed {
+    /// @notice Emitted when a withdrawal is finalized
+    event WithdrawalFinalized(bytes32 indexed);
+
+    /// @notice Value used to reset the l2Sender, this is more efficient than setting it to zero.
+    address internal constant DEFAULT_L2_SENDER = 0x000000000000000000000000000000000000dEaD;
```

Comment:
This could use https://github.com/ethereum-optimism/optimism/blob/83ce16a00bb4100f504c5726aa5362e92077a758/packages/contracts/contracts/libraries/constants/Lib_DefaultValues.sol#L11

Reply:
It could but for a single var, IMO it just adds one more place for the reader to jump to.